programs/pluto/kernel_xfrm.c

```diff
@@ -515,6 +561,9 @@ static bool sendrecv_xfrm_msg(struct nlmsghdr *hdr,
 		 */
 		ldbg(logger, "%s() netlink response for %s %s included non-error error",
 		     __func__, description, story);
+		LDBGP_JAMBUF(DBG_BASE, logger, buf) {
```

Comment:
... this code changed to:
```
if (DBGP(DBG_BASE)) {
    LDBG_log(logger, "%s() netlink response for %s %s included non-error error",
		     __func__, description, story);
   llog_ext_ack(DEBUG_STREAM, logger, &rsp.n);
}

Reply:
PS: you could even merge the "netlink response for ..." `into llog_ext_ack()`

Reply:
Done.

> PS: you could even merge the "netlink response for ..." into llog_ext_ack()

I left it as-is, as it might make testing a little harder (some tests are expecting this line and extack might not be available if kernel is older.